<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultisyncForm</class>
 <widget class="QWidget" name="MultisyncForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="tab_stw">
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QGridLayout">
         <item row="0" column="2">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="tab_name">
           <property name="statusTip">
            <string>Set multisync name</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>250</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_21">
               <property name="pixmap">
                <pixmap resource="../resources.qrc">:/new/prefix1/images/sources.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>&lt;b&gt;Sources:&lt;/b&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QListWidget" name="list_multi">
               <property name="statusTip">
                <string>List of multisync sources</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QPushButton" name="add_multi">
                   <property name="statusTip">
                    <string>Add source</string>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                   <property name="icon">
                    <iconset resource="../resources.qrc">
                     <normaloff>:/new/prefix1/images/add.png</normaloff>:/new/prefix1/images/add.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="remove_multi">
                   <property name="statusTip">
                    <string>Remove source</string>
                   </property>
                   <property name="text">
                    <string>Remove</string>
                   </property>
                   <property name="icon">
                    <iconset resource="../resources.qrc">
                     <normaloff>:/new/prefix1/images/remove.png</normaloff>:/new/prefix1/images/remove.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="vars_multi">
                   <property name="statusTip">
                    <string extracomment="Modify the variables for the paths of Multisync sources">Modify source path variables</string>
                   </property>
                   <property name="text">
                    <string>Variables</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_22">
               <property name="pixmap">
                <pixmap resource="../resources.qrc">:/new/prefix1/images/Synkron16.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>&lt;b&gt;Destination:&lt;/b&gt;</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="destination_multi">
               <property name="statusTip">
                <string>Set a destination</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="browse_multi">
               <property name="statusTip">
                <string>Browse for destination</string>
               </property>
               <property name="text">
                <string>Browse</string>
               </property>
               <property name="icon">
                <iconset resource="../resources.qrc">
                 <normaloff>:/new/prefix1/images/folder_16.png</normaloff>:/new/prefix1/images/folder_16.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="ExtendedLineEdit" name="search_multi">
             <property name="statusTip">
              <string>Search sync log</string>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="logs_stw">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_2">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTableWidget" name="tw_multi">
                 <property name="statusTip">
                  <string>Multisync log</string>
                 </property>
                 <property name="columnCount">
                  <number>2</number>
                 </property>
                 <column/>
                 <column/>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_3">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="ExtendedTreeWidget" name="analyse_tree">
                 <column>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
          <zorder>logs_stw</zorder>
          <zorder>search_multi</zorder>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="1">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="sync_multi">
           <property name="statusTip">
            <string>Start multisync</string>
           </property>
           <property name="text">
            <string>Multisync</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/new/prefix1/images/multisync.png</normaloff>:/new/prefix1/images/multisync.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="stop_sync_btn">
           <property name="statusTip">
            <string>Stop multisync</string>
           </property>
           <property name="text">
            <string>Stop multisync</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/new/prefix1/images/stop_sync22.png</normaloff>:/new/prefix1/images/stop_sync22.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="go_to_analyse">
           <property name="statusTip">
            <string>Analyse</string>
           </property>
           <property name="text">
            <string>Analyse</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/new/prefix1/images/analyse_16.png</normaloff>:/new/prefix1/images/analyse_16.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="show_advanced">
           <property name="statusTip">
            <string>Show advanced options of configuration</string>
           </property>
           <property name="text">
            <string>Advanced</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/new/prefix1/images/configure16.png</normaloff>:/new/prefix1/images/configure16.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="advanced_layout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>extendedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ExtendedTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>extendedtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
